### PR TITLE
Fix way to include CSS3 syntax fo neovim

### PR DESCRIPTION
**Pull request by vwxyutarooo**
Neovim does have different folder structure. I'm not sure is this make sense? Otherwise I'm gonna make other commit to disable for neovim totally.

### CODE REVIEW

**Comment by fleischie** [comment]
Hey @vwxyutarooo thanks for your contribution.

I may have a question though: The change you propose is the same as the previous line in that file, just with an `nvim`-guard. You are basically importing the default CSS definitions as `@CSS` and again as `@CSS3`. The block you are referring to is specifically for the recommended CSS3 syntax plugin from the README.md file ([hail2u/vim-css3-syntax](https://github.com/hail2u/vim-css3-syntax)): That particular plugin uses a subfolder structure instead of the default one-file syntax definitions.

Basically I wonder whether there is a very specific problem you are expriencing, and that this proposed change fixes?

Also neovim and classic vim have the same folder structure of where the syntax items reside in, the runtime path just differs (it's both `syntax/css.vim`, but once `<path-to-nvim>/<version-number>/share/nvim/runtime/syntax/css.vim` and once `<path-to-vim>/vim<version>/syntax/css.vim`).

If you have any more insights into this particular change you are suggesting, please don't hesitate to ask. :v: :smile:

**Comment by vwxyutarooo** [comment]
> You are basically importing the default CSS definitions as @CSS and again as @CSS3

Okay, I got it.

> Basically I wonder whether there is a very specific problem you are expriencing, and that this proposed change fixes?

Yeah you are right actually. I got something error from that line. However, somehow it won't now.
I close this P-R. Thanks.
